database/setup_test.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 func TestNewDatabase(t *testing.T) {
-	testSampleInvalidDb, _ := filepath.Abs(filepath.Join("testdata", "invaliddb.go"))
+	testSampleInvalidDb, _ := filepath.Abs(filepath.Join("testdata", "invaliddb.db"))
```

Comment:
Hey, what you did here is the `file does not exist test`, which we already did. What we wanted to do is the invalid db file test. Invalid as in the file exists, but is not a sqlite database.

Reply:
I changed the testSampleInvalidDb to be the new invalid .db file. Isnt that right?

Reply:
So can I merge it or am I understanding this completely wrong?

Reply:
But isnt invaliddb.db a non existing file? Or did you forget to push it to the branch?

Reply:
Oh i see... i forgot to add the file to git :)

Reply:
It was probably ignored by gitignore. This seems good. We can merge this now :+1:

Reply:
@vyshnav-vinod pushed the file